src/app/productConfig/productConfig.spec.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import 'angular-mocks';
+import module from './cart.component.js';
```

Comment:
Running karma locally, this is a 404. Idk why this causes Phantom to crash but hopefully updating that fixes it...